workspaces/global-floating-action-button/plugins/global-floating-action-button/src/components/FloatingButton.tsx

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2025 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as React from 'react';
+import classnames from 'classnames';
+
+import { makeStyles } from '@mui/styles';
+import { FlexDirection, FloatingActionButton, Slot } from '../types';
+import { FAB, FABWithSubmenu } from './FAB';
+import { useLocation } from 'react-router-dom';
+import { filterAndSortButtons } from '../utils';
+
+const useStyles = makeStyles(() => ({
+  button: {
+    zIndex: 200,
+    display: 'flex',
+    position: 'fixed',
+    maxWidth: '150px',
+    gap: '10px',
+  },
+  'page-end': {
+    bottom: '60px',
+    right: '60px',
+  },
+  'bottom-center': {
+    bottom: '60px',
+    left: '50%',
+  },
+}));
```

Comment:
You could use the theme, like:

```suggestion
const useStyles = makeStyles((theme) => ({
  button: {
    zIndex: 200,
    display: 'flex',
    position: 'fixed',
    maxWidth: '150px',
    gap: '10px',
  },
  'page-end': {
    bottom: theme.spacing(4),
    right: theme.spacing(4),
  },
  'bottom-center': {
    bottom: theme.spacing(4),
    left: '50%',
  },
}));
```